patroni/utils.py

```diff
@@ -357,6 +357,8 @@ def polling_loop(timeout, interval=1):
 
 def split_host_port(value, default_port):
     t = value.rsplit(':', 1)
+    if ':' in t[0]:
```

Comment:
Whats the value of this `if`

Reply:
It is about IPv6 format, for example `::1`.